common/transport/http/test/_rest_api_client_test.js

```diff
@@ -113,6 +113,23 @@ describe('RestApiClient', function() {
       client.executeApiCall('GET', '/fake/path', null, null, testCallback);
     });
 
+    /* Tests_SRS_NODE_IOTHUB_REST_API_CLIENT_18_002: [ If an `x509` cert was passed into the constructor via the `config` object, `executeApiCall` shall use it to establish the TLS connection. ] */
+    it('uses the x509 cert if passed', function(testCallback) {
+      var fakeHttpHelper = {
+        buildRequest: function(method, path, headers, host, x509, requestCallback) {
+          assert.equal('yes', x509.fake);
```

Comment:
>equal [](start = 17, length = 5)

we try to avoid .equal() now and instead use either .strictEqual() or .deepEqual(). In that case I believe you can just declare the fakeConfig object first, and just say assert.strictEqual(x509, fakeConfig.x509)